packages/superset-ui-connection/test/types.ts

```diff
@@ -0,0 +1 @@
+declare module 'fetch-mock';
```

Comment:
Move this file to `packages/superset-ui-connection/types/external.d.ts` should resolve the lint error in CI.